### ISSUE TITLE
Themenabend: Bringing Vulkan to life - DXVK

### ISSUE THREAD

**Issue by Lazalatin** [foss-ag/website_hack-and-snack-public#1]
Bei diesem Themenabend möchte ich mich für eine Stunde der Vulkan API widmen und dabei einige Konzepte anreißen welche hinter Vulkan stecken und inwiefern es sich von seinem Vorgänger OpenGL unterscheidet.

Bei dem Abend möchte ich insbesondere Bezug nehmen auf das Projekt https://github.com/doitsujin/dxvk von @doitsujin, welches mir persönlich schon oft zu Nutzen kam.

@doitsujin: Falls du dies liest: Ich würde mich freuen dich bei diesem Themenabend als Gast begrüßen zu dürfen und es wäre uns von der FOSS-AG Dortmund eine Ehre, wenn du einen Kurzvortrag zu der Historie zu DXVK halten und deine Ideen und Motivation dahinter beleuchten könntest. Anschließend gäbe es einen gemütlichen Ausklang des Abends mit kostenlosen Getränken. Natürlich wäre ich dir bei der Reiseplanung und Unterbringung behilflich und beantworte dir gerne alle Fragen.

@FOSSler: Gibt es Ideen, Wünsche oder Hinweise zu dieser Planung? Kennt ihr eventuell noch wen mit OpenGL oder Vulkan-Expertise?

**Comment by ROYALBEFF** [comment]
Kenne mich nicht wirklich mit Vulkan aus, aber da Vulkan die Zukunft ist, wäre ein dementsprechender Themenabend ziemlich cool. Würdest du das im Juli machen wollen?

**Comment by Lazalatin** [comment]
Nein, soweit ich weiß ist da noch ein anderes Thema in der Pipeline.

Ich würde gerne erst einmal abwarten ob @doitsujin hierauf reagiert. Wenn ich das nämlich alleine vorbereiten muss/sollte, werde ich definitiv länger brauchen... :smile:

**Comment by ROYALBEFF** [comment]
Wüsste zwar gerade nicht, dass noch was in der Queue ist, aber ist auch nicht weiter schlimm, wenn das Thema einen Monat später dran ist.
